### PR TITLE
User Publication + Subscription + Example React Components

### PR DESCRIPTION
**Pull request by mhoc**

### CODE REVIEW

File: imports/ui/views/users/UsersList/index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import ULSC from './UsersListStateContainer'
```
Comment (garr741) [inline imports/ui/views/users/UsersList/index.js]: MALTWGC (Maybe abbreviations like this will get confusing?) 
Why not just use the full name?

**Review by garr741 — APPROVED** [review]

File: imports/ui/views/users/UsersList/index.js (hunk 1 above)
Comment (mhoc) [reply imports/ui/views/users/UsersList/index.js]: Laziness.

File: imports/ui/views/users/UsersList/UsersList.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { CircularProgress } from 'material-ui'
+import React, { PropTypes } from 'react'
+
+const styles = {
+    firstName: {
```
Comment (bwencke) [inline imports/ui/views/users/UsersList/UsersList.jsx:5]: 4 spaces instead of 2?

File: imports/ui/views/users/UsersList/index.js (hunk 1 above)
Comment (bwencke) [reply imports/ui/views/users/UsersList/index.js]: Agreed

**Pull request merged**